uaclient/status.py

```diff
@@ -200,6 +215,7 @@ class UserFacingStatus(enum.Enum):
 Personal and community subscriptions are available at no charge
 See https://ubuntu.com/advantage"""
 MESSAGE_ENABLE_BY_DEFAULT_TMPL = "Enabling default service {name}"
+MESSAGE_ENABLE_REBOOT_REQUIRED = "A reboot is required to complete install"
```

Comment:
This is going to be true on disable as well. Just another variable or maybe make it "MESSAGE_REBOOT_REQUIRED"?

Reply:
Guess they're slightly different message strings aren't they. Ignore me.

Reply:
yeah next branch will make this message a template where we'll print "install" vs "disable process"